examples/trianglell/main.rs

```diff
@@ -375,9 +376,10 @@ fn main() {
             }
         }
 
-        graphics_pool.reset();
+        general_queues[0].wait_idle(); // SLOW!
```

Comment:
Hm, not totally sure if this can be removed. The pipeline barrier enforces that the data has been uploaded to the gpu?
Most/All examples I've seen so far wait after the texture upload, wondering if this is just over-conservative.
(I'm still not feeling 100% safe when it comes to synchronization/ordering guarantees...)